src/App.tsx

```diff
@@ -1,61 +1,138 @@
+import React, { useEffect, useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { Todo, Error } from './types';
+
+const getUserById = (userId: number) => {
```

Comment:
Consider renaming `getUserById` to `findUserById` to better describe what the function does. 🤔